src/App.jsx

```diff
@@ -1,14 +1,45 @@
+import { useState } from 'react';
 import './App.css';
+import messages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+
+
 
 const App = () => {
+
+  //set the initial state for the msgs
+  const [entries, setEntries] = useState(messages);
```

Comment:
👍 Nicely done! We should be set state to all the messages so that `App` knows when a single message changes and can re-render the page.